compiler/src/dmd/mars.d

```diff
@@ -118,7 +117,15 @@ Where:
 %.*s", cast(int)inifileCanon.length, inifileCanon.ptr, cast(int)help.length, &help[0]);
 }
 
-extern (C++) void generateJson(ref Modules modules)
+/*******************************************
+ * Generate JSON file.
+ * Params:
+ *      modules = Modules
+ *      eSink = error message sink
+ * Returns:
+ *      true on error
+ */
+extern (C++) bool generateJson(ref Modules modules, ErrorSink eSink)
```

Comment:
this is forward declared in LDC, but unused, and not in a header. 
fyi @ibuclaw 

Does this still need to be `extern(C++)`?